index.html

```diff
@@ -0,0 +1 @@
+<h1>Hello world!</h1>
```

Comment:
Would be great to add basic html, head, body etc tags
https://www.w3schools.com/html/html_basic.asp